vajram/vajram-codegen/src/main/java/com/flipkart/krystal/vajram/codegen/VajramCodeGenerator.java

```diff
@@ -288,11 +288,9 @@ private ParsedVajramData getParsedVajramData() {
     return Optional.ofNullable(parsedVajramData).orElseGet(this::initParsedVajramData);
   }
 
-  private static ImmutableSet<String> getResolverSources(ExecutableElement resolve) {
+  private static ImmutableSet<String> getResolverSources(ExecutableElement resolve, Utils util) {
```

Comment:
I would suggest make this method non-static and remove this util parameter.

Reply:
Sure, will make the change